src/__forks__/createRequestError.js

```diff
@@ -0,0 +1,10 @@
+import formatRequestErrors from './formatRequestErrors';
+
+export default function createRequestError(request, requestType, payload) {
```

Comment:
this isn't actually a fork, right? you just rewrote it? better to move this into `NetworkLayer.js` i think